src/components/filter/SliderFilter.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { Slider } from 'react-native-elements';
+
+import { colors, normalize } from '../../config';
+import { Label } from '../Label';
+import { RegularText } from '../Text';
+import { WrapperRow } from '../Wrapper';
+
+type Props = {
+  label?: string;
+  onValueChange: (value: number) => void;
+  thumbStyle?: ViewStyle;
+  value: number;
+};
+
+export const SliderFilter = ({ label, onValueChange, thumbStyle, value, ...props }: Props) => {
+  return (
+    <>
+      <Label>{label}</Label>
+      <WrapperRow spaceBetween style={styles.alignItemsCenter}>
+        <Slider
+          maximumValue={100}
+          onValueChange={onValueChange}
+          step={10}
```

Comment:
the max value and steps should be define by the values we receive from the server configs, right?

Reply:
Yes, that's right

Reply:
i see values with 1, 11, 21, ..., 71, ... is this correct?

![Simulator Screen Recording - iPhone 15 Plus - 2024-09-27 at 16 38 11](https://github.com/user-attachments/assets/8b99a32f-4676-4235-9c80-f8bce3b8637f)

and: it is very laggy using the slider! does it work smoothly for you?

Reply:
i found out, that every sliding re-renders all the filters instead of just the slider, you can see it in the resulting console warnings here:

![Simulator Screen Recording - iPhone 15 Plus - 2024-09-27 at 18 38 47](https://github.com/user-attachments/assets/30e61df8-59ac-4c3c-bc93-3481fc3161f1)

this needs to be tackled.

Reply:
I solved the problem.